controllers/admin/AdminShippingController.php

```diff
@@ -43,72 +43,72 @@ public function __construct()
                         'cast' => 'floatval',
                         'type' => 'text',
                         'validation' => 'isPrice'),
                     'PS_SHIPPING_FREE_PRICE' => array(
-                        'title' => $this->l('Free shipping starts at'),
+                        'title' => $this->trans('Free shipping starts at', array(), 'Admin.Shipping.Feature'),
                         'suffix' => $this->context->currency->getSign(),
                         'cast' => 'floatval',
                         'type' => 'text',
                         'validation' => 'isPrice'),
                     'PS_SHIPPING_FREE_WEIGHT' => array(
-                        'title' => $this->l('Free shipping starts at'),
+                        'title' => $this->trans('Free shipping starts at', array(), 'Admin.Shipping.Feature'),
                         'suffix' => Configuration::get('PS_WEIGHT_UNIT'),
                         'cast' => 'floatval',
                         'type' => 'text',
                         'validation' => 'isUnsignedFloat'),
                 ),
                 'description' =>
                     '<ul>
-						<li>'.$this->l('If you set these parameters to 0, they will be disabled.').'</li>
-						<li>'.$this->l('Coupons are not taken into account when calculating free shipping.').'</li>
+						<li>'.$this->trans('If you set these parameters to 0, they will be disabled.', array(), 'Admin.Shipping.Feature').'</li>
+						<li>'.$this->trans('Coupons are not taken into account when calculating free shipping.', array(), 'Admin.Shipping.Feature').'</li>
 					</ul>',
                 'submit' => array('title' => $this->trans('Save', array(), 'Admin.Actions'))
             ),
             'general' => array(
-                'title' => $this->l('Carrier options'),
+                'title' => $this->trans('Carrier options', array(), 'Admin.Shipping.Feature'),
                 'fields' => array(
                     'PS_CARRIER_DEFAULT' => array(
-                        'title' => $this->l('Default carrier'),
-                        'desc' => $this->l('Your shop\'s default carrier'),
+                        'title' => $this->trans('Default carrier', array(), 'Admin.Shipping.Feature'),
+                        'desc' => $this->trans('Your shop\'s default carrier', array(), 'Admin.Shipping.Help'),
                         'cast' => 'intval',
                         'type' => 'select',
                         'identifier' => 'id_carrier',
                         'list' => array_merge(
                             array(
-                                -1 => array('id_carrier' => -1, 'name' => $this->l('Best price')),
-                                -2 => array('id_carrier' => -2, 'name' => $this->l('Best grade'))
+                                -1 => array('id_carrier' => -1, 'name' => $this->trans('Best price', array(), 'Admin.Global')),
```

Comment:
Both "Best price" and "Best grade" should stay Admin.Shipping.Feature as it's related to shipping and not appearing anywhere else.